app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] After fixing the Devise issue above remember to ensure the login works as well.

- [ ] Kindly ensure your page layout matches the wireframe.

  wireframe | your layout
  :---:|:---:
  ![login](https://user-images.githubusercontent.com/36542986/194023231-9d35d614-05a5-4bf5-8a16-86eaf8b80a55.jpg) | ![login-1](https://user-images.githubusercontent.com/36542986/194023226-30dad8c6-8f2c-4423-85f2-d918eba6531f.jpg)